O/oneAPI_Support/build_tarballs.jl

```diff
@@ -178,7 +178,7 @@ done
 mkdir -p ${libdir} ${includedir}
 cp -r include/* ${includedir}
 for lib in sycl svml irng imf intlc ur_loader ur_adapter \
-           mkl_core mkl_intel_ilp64 mkl_sequential mkl_sycl \
+           mkl_cdft_core mkl_core mkl_intel_ilp64 mkl_sequential mkl_sycl \
            mkl_avx mkl_def umf tcm; do
     cp -a lib/lib${lib}*.so* ${libdir}
```

Comment:
Using verbose commands is a bit nicer, to keep record in the log of what actually happened
```suggestion
    install -Dvm 755 lib/lib${lib}*.so* -t ${libdir}
```